stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/ChannelNameFormatter.kt

```diff
@@ -43,14 +43,17 @@ public fun interface ChannelNameFormatter {
          *
          * @param context The context used to load string resources.
          * @param fallback The resource identifier of a fallback string.
+         * @param maxMembers The maximum number of members used to generate a name for a distinct channel.
          *
          * @see [DefaultChannelNameFormatter]
          */
+        @JvmOverloads
         public fun defaultFormatter(
             context: Context,
             @StringRes fallback: Int = R.string.stream_ui_channel_list_untitled_channel,
+            maxMembers: Int = 5
```

Comment:
Hey @leandroBorgesFerreira,

I'm wondering why the number `5` is set as default here. 
Is that because of some of our docs where this number is mentioned?

Reply:
We probably don't want to process more than X number of names for the name, as 3-5 usually take up enough space to overflow into an ellipsis, so it's a bit wasteful to format for e.g. 30+ (or even hundreds).